scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+echo "File Submission Started."
```

Comment:
Change this to `Submitting file to EvalAI...`

Reply:
Done.